### ISSUE TITLE
Spotify playback- widevine issue?

### ISSUE THREAD

**Issue by ajarang** [ungoogled-software/ungoogled-chromium-windows#130]
I am running 97.0.4692.99, windows 10. I installed this build from a prebuilt (macchrome)[https://github.com/macchrome/winchrome/releases]  Spotify keeps randomly skipping tracks after playing them for a split second. From my limited research, I am lead to believe that this is a widevine issue, since I installed google chrome and the issue went away.

**Comment by tomasz1986** [comment]
Just a comment, but you may want to switch to the officially approved builds listed at https://ungoogled-software.github.io/ungoogled-chromium-binaries. Anything else is treated as 3rd party and issues reported with such builds may be dismissed.

Of course, if the exact same problem occurs in the official build, then it's a different story 🙂.

**Comment by ajarang** [comment]
Hmm. I should've added that I was using winget to download them, so I assumed it was from the official build. Bit weird that both 'winget' and 'choco' installs point to different sources. I tried uninstalling and downloading from the link that you have posted above, and the issue persists.

**Comment by networkException** [comment]
> Just a comment, but you may want to switch to the officially approved builds listed at https://ungoogled-software.github.io/ungoogled-chromium-binaries. Anything else is treated as 3rd party and issues reported with such builds may be dismissed.

That is wrong, the readme declares binaries from *that* page as third party. The term official is not agreed on at the moment (see #1549)

**Comment by ajarang** [comment]
still having this issue... can anyone test on windows 10 to see if it is reproducible?

**Comment by networkException** [comment]
Widewine is proprietary and as such supporting it is quite hard. I'm unaware of a case in which widewine works with the current set of patches

**Comment by ajarang** [comment]
I'm not sure if it is a widevine issue, which is why I wanted to see if it was reproducible elsewhere. This wasn't an issue until I updated, and the issue persists with the latest version.
